api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -293,17 +296,17 @@ public void testCreateAccount_noRequireInvitationKey() {
 
   @Test(expected = BadRequestException.class)
   public void testInvitationKeyVerification_invitationKeyMismatch() {
+    invitationVerificationRequest.setInvitationKey("wrong key");
```

Comment:
the logic of this test was backwards until I moved the invitiation key mock to where it always ran.

Previously: called `invitationKeyVerification()` with the correct key, which didn't match the mock which *didn't* return the correct key

Now: mock returns correct key, and this test attempts to verify a wrong key.